File: browser-use-node/examples/basic.ts
/**
 * Basic task — run a task and get the result.
 */
import "dotenv/config";
import { BrowserUse } from "browser-use-sdk/v4";

async function main() {
  const client = new BrowserUse();

  const run = await client.runs.create({
    task: "Search for the top 10 Hacker News posts and return the title and URL.",
  });
  const result = await client.runs.waitForCompletion(run.id);
  console.log(result.result);
}

await main();

File: browser-use-python/examples/basic.py
from dotenv import load_dotenv
from browser_use_sdk.v4 import BrowserUse

load_dotenv()

with BrowserUse() as client:
    run = client.runs.create(
        "Search for the top 10 Hacker News posts and return the title and URL."
    )
    result = client.runs.wait_for_completion(run.id)
    print(result.result)
